t/proxyunits.t

```diff
@@ -116,11 +135,12 @@ for my $msrv (@mocksrvs) {
     is(scalar <$ps>, "ok\r\n", "got data back");
     is(scalar <$ps>, "END\r\n", "got end string");
 
-    like(<$w>, qr/ts=(\S+) gid=\d+ type=proxy_backend error=trailingdata name=127.0.0.1 port=\d+ depth=0 rbuf=garbage/, "got backend error log line");
+    like(<$w>, qr/ts=(\S+) gid=\d+ type=proxy_backend error=trailingdata name=127.0.0.1 port=\d+ depth=0 rbuf=garbage/, "got proxy event log line");
```

Comment:
proxy event is a stream; "backend error" is referring to the `type=proxy_backend` here

Reply:
ack